pulsar-client-shaded/pom.xml

```diff
@@ -0,0 +1,140 @@
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.apache.pulsar</groupId>
+    <artifactId>pulsar</artifactId>
+    <version>1.21.0-incubating-SNAPSHOT</version>
+    <relativePath>..</relativePath>
+  </parent>
+
+  <artifactId>pulsar-client</artifactId>
```

Comment:
I am trying to understand need of this pom. can we avoid creating this additional pom and mention shaded artifact file name in [plugin](http://maven.apache.org/plugins-archives/maven-shade-plugin-1.4/shade-mojo.html#finalName) it self?

Reply:
I just tried this [commit](https://github.com/rdhabalia/pulsar/commit/e0429c2f21f8ab341e58f4257adfa38a8aa222ae) it creates `pulsar-client.jar`(shaded jar) and `pulsar-client-original.jar` (actual jar without dep). So, should we avoid creating this pom file.